website/views/options/settings.jade

```diff
@@ -379,12 +379,12 @@ script(id='partials/options.settings.subscription.html',type='text/ng-template')
 
           .form-inline
             .form-group
-              input.form-control(type='text', ng-model='_subscription.coupon', placeholder='Promo Code')
+              input.form-control(type='text', ng-model='_subscription.coupon', placeholder= env.t('couponPlaceholder'))
```

Comment:
I'm not sure promo code and coupon code are the same thing:

### Before
![screen shot 2015-05-03 at 2 32 55 pm](https://cloud.githubusercontent.com/assets/2916945/7446543/5fa8060a-f1a1-11e4-9626-242bf8e92054.png)

### After
![screen shot 2015-05-03 at 2 34 01 pm](https://cloud.githubusercontent.com/assets/2916945/7446546/767143ce-f1a1-11e4-835b-e85d61678172.png)

@lemoness Do you have thoughts on this? Is it okay to use coupon language here instead of promo?